gammapy/stats/variability.py

```diff
@@ -228,3 +230,101 @@ def compute_flux_doubling(flux, flux_err, coords, axis=0):
     }
 
     return doubling_dict
+
+
+def TKAlgorithm(
+    power_spectrum,
+    npoints,
+    spacing,
+    type="discrete",
+    random_state="random-seed",
+    normalization=1.0,
+):
+    """Implementation of the Timmer-Koenig algorithm to simulate a time series from a power spectrum.
+
+    Parameters
+    ----------
+    power_spectrum : float, `~numpy.ndarray`
+        Power spectrum used to generate the time series. It can either be presented analytically, as a parameter or set
```

Comment:
It is unclear how to pass the `power_spectrum` analytically. What are the parameters and how are they ordered?

Reply:
Is it accepted to have sub-indentations in the docstrings? That would be the simplest way to add the parameter explanations for each supported analytical shape.